wof/apps/spyned_1_1.py

```diff
@@ -209,7 +209,7 @@ def GetValuesForASiteObject(ctx, site, startDate=None, endDate=None, authToken=N
             try:
                 timeSeriesResponse = wof_inst.create_get_values_site_response(
                         site, startDate, endDate)
-                outStream = io.BytesIO()
+                outStream = io.StringIO()
```

Comment:
We should document that stream is expected to be `StringIO`! When dealing with streamed data I prefer `BytesIO` but I am 100% on Python 3 now so I don't have the problems we are seeing here :smile:

Reply:
I wish I can be 100% Python 3 😛, I wouldn't need to deal with the unicode craziness! lol

Reply:
The number of "non-ASCII compliant languages" in ODM2 is quite high to justify moving to Python 3, no? :wink: 

If language is not a strong argument think of the cute symbols and greek letters we can use :smile:

Reply:
> The number of "non-ASCII compliant languages" in ODM2 is quite high to justify moving to Python 3, no? 

It's all up to the bosses 😉  haha

> If language is not a strong argument think of the cute symbols and greek letters we can use 

Oh man! I would use more emojis in code in that case! 👍